esx_service/utils/kvESX.py

```diff
@@ -175,24 +185,26 @@ def load(volpath):
     except IOError:
         return None
 
-    kvDict = json.load(fh)
-
+    kv_str = fh.read()
```

Comment:
Thanks I'll post the changes
On Jun 8, 2016 9:31 PM, "Mark Sterin" <notifications@github.com> wrote:

> In esx_service/utils/kvESX.py
> <https://github.com/vmware/docker-volume-vsphere/pull/433#discussion_r66284803>
> :
>
> > @@ -175,24 +185,26 @@ def load(volpath):
> >      except IOError:
> >          return None
> >
> > -    kvDict = json.load(fh)
> > -
> > +    kv_str = fh.read()
>
> if read throws. this will leave file open
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/433/files/172872ed7d4b48ab510cb3df533bba4d0db79b68#r66284803>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseCPAC4ltRdZ8yHCI-fiSFgGYgjAEks5qJucagaJpZM4IrROI>
> .
>